package.json

```diff
@@ -36,7 +36,8 @@
     "ssb-crut": "^2.1.2",
     "ssb-keys": "^8.0.2",
     "ssb-ref": "^2.14.3",
-    "ssb-schema-definitions": "^3.1.0"
+    "ssb-schema-definitions": "^3.1.0",
+    "tap-spec": "^5.0.0"
   },
   "devDependencies": {
     "faucet": "0.0.1",
```

Comment:
gotta remove faucet if you're not using it!

Reply:
i'll remove this, i just used tape because faucet wasnt what i was used to

Reply:
(was supposed to remove it before pushing anyway)

Reply:
Yeah faucet hides console.log output, which is good sometimes and annoying sometimes